extra/exports

```diff
@@ -12,6 +12,7 @@
 base64_bufsize
```

Comment:
> Co-authored-by: Nikita
> Pettik
> <korablev@tarantool.org>

Looks like your editor is broken, this should be on a single line.

Reply:
Fixed.